.gitlab-ci.yml

```diff
@@ -12,5 +17,107 @@ include:
       - '.gitlab-ci-check-commits.yml'
       - '.gitlab-ci-check-license.yml'
 
+test:
```

Comment:
Let's remove this and rely only on the template - we don't need coverage from this tool from the acceptance tests